webpack.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    entry: ["./public/scripts/history.js", "./public/scripts/confetti.js", "./public/scripts/main.js","./public/scripts/keymaster.js" ],
+    target: 'node',
```

Comment:
Based on a quick read of the [webpack config docs](https://webpack.js.org/configuration/target/) I think we would want to set this to `browser` instead of `node`.